### DISCUSSION TITLE
[Feature Request] Update Slider Tooltip

### DISCUSSION THREAD

**Discussion by CodeVisionEvgen** [heroui-inc/heroui#4741] [Feature requests]
### Is your feature request related to a problem? Please describe.

I recommend adding a new renderTooltip parameter that has the value Slider as the first argument.
Example:

```
  const formatTime = (time: number) => {
    const minutes = Math.floor(time / 60);
    const seconds = Math.floor(time % 60);
    return `${minutes}:${seconds < 10 ? '0' : ''}${seconds}`;
  };
...
...
<Slider showToolTip renderTooltip={(value)=> <p>{formatTime(value)}</p>}/>
```

### Describe the solution you'd like

none

### Describe alternatives you've considered

none

**Comment by linear** [comment]
<p><a href="https://linear.app/nextui-inc/issue/ENG-1136/[feature-request]-update-slider-tooltip">ENG-1136 [Feature Request] Update Slider Tooltip</a></p>

**Comment by pratyush3124** [comment]
would like to work on this feature

**Comment by ararTP** [comment]
Any news/ workarounds?
one workaround would be to use :
```jsx
tooltipProps={{
    content: formatTime(stateValue)
}}
```
but it won't work on multiple thumbs
